config/locales/en.yml

```diff
@@ -1169,18 +1174,18 @@ en:
       edit:
         confirm_course: Confirm course
         edit_details: Edit details for this trainee only
-    lead_schools:
+    lead_partners:
       edit:
-        heading: Lead school
-        hint: Search for a school by its unique reference number (URN), name or postcode
-        description: The lead school is the main organisation and point of contact for training providers, placements and partner schools in the school direct partnership.
+        heading: Lead partner
```

Comment:
![image](https://github.com/user-attachments/assets/b562b1dd-b0f5-4575-a73a-19f413308364)

1 needs adjusting

2 & 3 the validation message don't really tie up
4 & 5 is it still correct referring it as `school`?
6 seems a bit error prone given that there are fee-funded routes in play

probably need a bit of s steer from @andrewscrivener

Reply:
We've just had a content review for this. There is another ticket in the backlog to see whether a full rewrite will be worthwhile. Apart from the tab title which needs changing the other points are still currently correct so leave as is for the time being please.